frontend/src/views/Auth/Login/Login.js

```diff
@@ -24,8 +28,37 @@ class Login extends Component {
     });
   }
 
+  handleLogin = async (e) => {
+    e.preventDefault();
+    const { username, password } = this.state.data;
+
+    const validationMsg = UserService.validateLogin(username, 
+      password);
+
+    if (validationMsg !== '') {
+      console.log(validationMsg);
+      return;
+    } 
+
+    const { home } = routePaths;
+    const { success, data: error } = await UserService.login(
+      username,
+      password
+    );
+
+    if (success) return this.props.history.replace(home);
+    console.log(error.response.data);
+  }
+
+  handleChange = ({ currentTarget: input }) => {
```

Comment:
Is **important** to standardize the _form of method_, so far, the standard is: `function name_of_function(param...)`.

We can use this form inside function to be used for function, not for class scope functions.

Reply:
This is not resolved.